apps/server/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"baseUrl": ".",
```

Comment:
Feel like I've asked this before and you provided a useful reason to keep baseUrl, but the tsconfig docs say this is deprecated. Its used in NextJS but otherwise I'm pretty sure we should remove.

https://www.typescriptlang.org/tsconfig/#baseUrl
> This feature was designed for use in conjunction with AMD module loaders in the browser, and is not recommended in any other context.

Reply:
`baseUrl` removed as it's unnecessary